src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { Filter } from '../types/Filter';
+
+interface Props {
+  onFilter: (filter: Filter) => void;
+  activeTodosCount: number;
+  currentFilter: Filter;
+}
+
+export const Footer: FC<Props> = ({
+  onFilter,
+  activeTodosCount,
+  currentFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosCount} {activeTodosCount === 1 ? 'item' : 'items'} left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
```

Comment:
recommend to use map here
for example:
const links = Object.entries(Filter);
{links.map(([key, value]) => (
<a
href={#/${value === Filter.All ? '' : ${value}}}
className={cn('filter__link', { selected: currentFilter === value })}
data-cy={FilterLink${key}}
key={key}
onClick={() => onChangeType(value)}
>
{key}

))}